buildenv/jenkins/common/pipeline-functions

```diff
@@ -321,14 +322,17 @@ def workflow(SDK_VERSION, SPEC, SHAS, OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO,
     // Set ghprbGhRepository and ghprbActualCommit for the purposes of Github commit status updates
     // Only for Nightly build
     // PR build will already have them set
+    GITHUB_SERVER = ''
     if (BUILD_IDENTIFIER.toLowerCase() == "nightly") {
         ghprbGhRepository = OPENJ9_REPO.substring(OPENJ9_REPO.indexOf('.com') +5, OPENJ9_REPO.indexOf('.git'))
         ghprbActualCommit = SHAS['OPENJ9']
+        GITHUB_SERVER = OPENJ9_REPO.substring(OPENJ9_REPO.indexOf('github.'), OPENJ9_REPO.indexOf('.com') +4)
```

Comment:
This is inside the nightly build if condition - should it be generically available for all types of builds?

Reply:
Good point. At the moment it is not used outside this context. But if #2836 goes through, it will need to be available for PR builds too. Will move it.